client.yml

```diff
@@ -280,6 +280,98 @@ paths:
               meta:
                 apiEnrollmentVersion: 0.0.1
                 apiVersion: 0.0.1
+  /controllers:
+    get:
+      security:
+      - ztSession: []
+      - oauth2:
+        - openid
+      description: Retrieves a list of controllers
+      tags:
+      - Controllers
+      summary: List CAs
+      operationId: listControllers
+      parameters:
+      - type: integer
+        name: limit
+        in: query
+      - type: integer
+        name: offset
+        in: query
+      - type: string
+        name: filter
+        in: query
+      responses:
+        "200":
+          description: A list of controllers
+          schema:
+            $ref: '#/definitions/listControllersEnvelope'
+        "400":
+          description: The supplied request contains invalid fields or could not be
+            parsed (json and non-json bodies). The error's code, message, and cause
+            fields can be inspected for further information
+          schema:
+            $ref: '#/definitions/apiErrorEnvelope'
+          examples:
+            application/json:
+              error:
+                args:
+                  urlVars: {}
+                cause:
+                  details:
+                    context: (root)
+                    field: (root)
+                    property: fooField3
+                  field: (root)
+                  message: '(root): fooField3 is required'
+                  type: required
+                  value:
+                    fooField: abc
+                    fooField2: def
+                causeMessage: schema validation failed
+                code: COULD_NOT_VALIDATE
+                message: The supplied request contains an invalid document
```

Comment:
wording sounds off in some ways. "The supplied document is invalid" or "contains an invalid field"? or meh...